### DISCUSSION TITLE
Problem with CC2500 and receiver Radiolink R8EF

### DISCUSSION THREAD

**Discussion by DmitriyPopovv** [pascallanger/DIY-Multiprotocol-TX-Module#809] [Q&A]
Good afternoon. I use a 328P chip and  CC2500 module, which has only pins - MOSI , SCLK , MISO , CHN.
Pins GD02 , GD00 , PA_EN , LNA_EN module does not have, according to the scheme they should be connected.
  /*  1 */  {PROTO_RLINK  ,   RLINK_AIR  , 0 , P_HIGH  , NO_AUTOBIND , 8 , 0x00000000 },

Problem - the receiver is attached, but there is no stable connection, it does not even work at a distance of 50 cm. Instruments show the transmitter is not producing power.
There may be a problem due to not connecting these pins.
Can this be fixed with tuning bits?

**Comment by pascallanger** [comment]
Can you post a picture of your cc2500 rf module ?

**Reply by DmitriyPopovv** [reply]
I also tried with such a module without an amplifier, this one is in receivers Radiolink R8EF, the situation has not changed.
There is a range, the work is not stable, it works like this:
There is a signal for 1 sec, then there is a file safe for 1 sec, there is a signal for 1 sec, then there is a file safe for 1 sec. And so constantly.
![EWRF3081-ST2-RF-module-2-4GHz-RF-Module-TI-CC2500-1mW-100meter-2-4-ghz-rf jpg_Q90](https://user-images.githubusercontent.com/22314583/212960812-c1e7f6e6-a253-46b6-9de2-6a8c613bcb85.jpg)

**Reply by DmitriyPopovv** [reply]
Made an assembly on STM32F103 - everything works fine without problems. Why might not work on AVR 328?

**Comment by DmitriyPopovv** [comment]
EWRF3081 SX2 RF module 2.4GHz RF Module TI-CC2500 

This module is used in the remote Radiolink T8FB - [EWRF3081](https://www.aliexpress.com/item/4000597705561.html?spm=a2g0o.productlist.main.1.4ff166d1639Zr5&algo_pvid=46057c6f-e349-408d-a1d9-d980e1f147c5&algo_exp_id=46057c6f-e349-408d-a1d9-d980e1f147c5-0&pdp_ext_f=%7B%22sku_id%22%3A%2210000003734916964%22%7D&pdp_npi=2%40dis%21USD%216.99%215.94%21%21%21%21%21%40212278f516739004601423066d0665%2110000003734916964%21sea&curPageLogUid=BJXN3mnBXLWk)

![image](https://user-images.githubusercontent.com/22314583/212762030-3683272a-e169-4192-bdc5-316f8eadfce9.png)
![image](https://user-images.githubusercontent.com/22314583/212762436-7bc1377e-bcdf-417a-8ca9-7bf6ad08cf6a.png)
